web-services/request.js

```diff
@@ -0,0 +1,13 @@
+//request.js
+//this function use https get method to get API data
+const https = require('https');
+function request(url){
```

Comment:
While it's great you were able to figure this out, you are "reinventing the wheel" here and missed the mark on this phase which was asking to "Research how to mock an API request in Jest". The mocking part is important because it is not advisable to make actual HTTP requests unless you are explicitly writing integration tests. Also, check for npm packages such as Axios, Supertest, etc. which encapsulate this type of functionality (and much more).